config.ini

```diff
@@ -1,5 +1,6 @@
 [settings]
-eventType=none
+eventType=birthday
 logging=true
 logFile=logs\log.txt
-overwriteSave=false
+overwriteSave=true
```

Comment:
The user should manually enable overwriting as they may have something they'd like to keep